### ISSUE TITLE
malformed utf-8 character and failed to load character config (errors 205 and 0)

### ISSUE THREAD

**Issue by bigsmxke** [kolton/d2bot-with-kolbot#1724]
Hi all, I'm getting the following errors. Uploading a snip of the error alongside a copy and paste of my config file. Not sure what exactly is going wrong. 

Many thanks in advance

![Capture](https://user-images.githubusercontent.com/52428254/60465577-55f25000-9c49-11e9-9448-5d47ada3dae2.PNG)
[config.txt](https://github.com/kolton/d2bot-with-kolbot/files/3347270/config.txt)

**Comment by Spoofzor** [comment]
The error occurs because it cannot load your class-personalized config file.

Found in  kolbot/libs/config
You should have it saved as Paladin.CharacterName.js or Sorceress.CharacterName.js

**Comment by bigsmxke** [comment]
> 
> 
> The error occurs because it cannot load your class-personalized config file.
> 
> Found in kolbot/libs/config
> You should have it saved as Paladin.CharacterName.js or Sorceress.CharacterName.js

Hi Spoofzor, 

Thanks, I had the filename saved as Sorceress.CharacterName instead of Sorceress.CharacterName.js. 
After I amended the name of the file, I attempted to run the bot and I no longer got the aforementioned error, but now it is reporting that the Attack config is incorrectly setup or something along those lines but I didn't look into it further as I was busy, so that's another thing I need to look into after work today, it's odd because I didn't have such an issue last ladder. 

I'll get on with it after work and come back in case of any issues. :) 

Many thanks, 
big
